system/handlers/admin/SiteTree.cfc

```diff
@@ -382,11 +383,16 @@ component extends="preside.system.base.AdminHandler" {
 		);
 
 		getPlugin( "MessageBox" ).info( translateResource( uri="cms:sitetree.pageEdited.confirmation" ) );
-
-		if ( _isManagedPage( page.parent_page, page.page_type ) ) {
-			setNextEvent( url=event.buildAdminLink( linkto="sitetree.managedChildren", querystring="parent=#page.parent_page#&pageType=#page.page_type#" ) );
+		cookie.sitetree_editPage_backToEdit = false;
+		if ( Val( event.getValue( name="_backToEdit", defaultValue=0 ) ) ) {
```

Comment:
We now prefer just using `rc` scope rather than `event.getValue()`. So:

```
if ( IsTrue( rc._backToEdit ?: "" ) ) {
// ...
}
```

(There is some older code that uses event.getValue(), but the elvis operator makes it cleaner to just use `rc`).